zk_toolbox/crates/zk_supervisor/src/main.rs

```diff
@@ -103,6 +105,7 @@ async fn run_subcommand(args: Supervisor, shell: &Shell) -> anyhow::Result<()> {
         }
         SupervisorSubcommands::Lint(args) => commands::lint::run(shell, args)?,
         SupervisorSubcommands::Fmt(args) => commands::fmt::run(shell.clone(), args).await?,
+        SupervisorSubcommands::ProtocolVersion => commands::protocol_version::run(shell).await?,
```

Comment:
probably prover protocol version? or it's a universal?
should we add a new layer?

Reply:
I mean... it's hard to say. It's more related to provers as they are using it, so yeah, I guess we can call it prover protocol version

Reply:
Just that I'm not sure that it needs a new layer, I don't have idea what else can be used for provers there. We can change the name to prover-version

Reply:
let's keep it prover version and later on will change it